reccmp/isledecomp/formats/mz.py

```diff
@@ -0,0 +1,74 @@
+from dataclasses import dataclass
+from pathlib import Path
+import struct
+
+from .image import Image
+
+
+class MZHeaderNotFoundError(ValueError):
+    """MZ magic string not found"""
+
+
+# pylint: disable=too-many-instance-attributes
+@dataclass(frozen=True)
+class ImageDosHeader:
```

Comment:
Not that we can't look it up (and not that this isn't a decades-old format) but it would be nice to have more descriptive member names here that are self-documenting. The ones in the PE module are a good example.

Reply:
It's a pythoninized version of the one in the C headers: `IMAGE_DOS_HEADER`
https://github.com/hughbe/windows-sdk-headers/blob/89f151d343587fcd5b854a4851fcb8f7187f3ac4/Include/10.0.19041.0/km/ntimage.h#L69

I'd prefer to keep it.